package.json

```diff
@@ -131,6 +131,7 @@
     "react-web-config": "^1.0.0",
     "save": "^2.4.0",
     "semver": "^7.3.8",
+    "setimmediate": "^1.0.5",
```

Comment:
Oh, I think this should be a dev dependency since it's only needed by jest?

Reply:
yeah makes sense, hope no one's gonna use it in the app though